index.html

```diff
@@ -43,6 +43,172 @@ <h3>LET'S CONNECT</h3>
         </section>
         
     </div>
+    <section class="works-section">
+        <section class="subsection-one">
+                <div>
+                    <img src="images/Snapshoot-Portfolio.png" alt="">
+                </div>
+                <div>
+                    <h3>
+                        Tonic
+                    </h3>
+                </div>
+                <div class="list-items">
+                    <h4>CANOPY</h4>
+                    <ul class="ul-list">
+                        <li>Back End Dev</li>
+                    </ul>
+                    <ul>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="paragraph">
+                    <h4>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h4>
+                </div>
+                <div class="btn">
+                    <button type="button" class="btn-1">html</button>
+                    <button type="button" class="btn-2">css</button>
+                    <button type="button" class="btn-3">javaScript</button>
+                </div>
+                <div class="big-btn">
+                    <button type="button" class="btn-4">See Project</button>
```

Comment:
- [ ] As for the `button` requirement, The button (e.g., "See project, get resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design). 
- 
![Screenshot from 2022-04-23 12-48-21](https://user-images.githubusercontent.com/26767383/164884564-722fb954-e945-4062-9984-24182a9c4759.png)

- **Hint** You can style on only one button and make the same class name to all the button present in your project.For tutorial , you can visit [here](https://www.w3schools.com/howto/howto_css_animate_buttons.asp)